src/components/Search/CompareTable.jsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@mui/material/IconButton';
+import Paper from '@mui/material/Paper';
+import { styled } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import store from '../../common/store';
+import { deleteRevision } from '../../reducers/SelectedRevisions';
+
+const StyledTableCell = styled(TableCell)(() => ({
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 14,
+  },
+  '&:first-of-type': {
+    width: '50px',
+  },
+  paddingTop: '10px',
+  paddingBottom: '10px',
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  '&:nth-of-type(odd)': {
+    backgroundColor: theme.palette.action.hover,
+  },
+}));
+
+function CompareTable(props) {
+  let { revisions } = props;
+
+  const selectedRevisions = [];
+  revisions?.forEach((item) => {
+    if (selectedRevisions.length === 0) {
+      selectedRevisions.push({ ...item, type: 'base' });
+    } else {
+      selectedRevisions.push({ ...item, type: 'new' });
+    }
```

Comment:
yeah so for this, we didnt decide how **base** will be re-assigned. if it was previously deleted.
Indexing it will actually cause more logic, I think once we decide the base re-assignment this will be clearer for me.